KPI/H.adoc

```diff
@@ -1,99 +1,168 @@
-== H. Data Parameter Keyword
+== H. Keywords
 
-Data parameter keywords are optional elements syntactically.  
-An xml record will validate without its presence.
+=== Status
 
+Draft
 
-=== H.1 To which WCMP element(s) does it apply ?
+=== WCMP element(s)
```

Comment:
Comment from my GISC Offenbach colleagues:
- descriptiveKeywords including attribute uuidref with following values: temporal, place, dataCentre, dataParam, theme
- using uuidref thesaurus subsection can be checked automatically
- where possible anchor to fixed codelists (like for theme WMO_CategoryCode or CI_RoleCode)
- use attribute codeListValue as mandatory (so that the text-value can be ignored)

Reply:
Hi @jsieland thanks for your comments, I didn't quite go through them before/during the meeting

- With regards to anchor comment , the KPI scores +1 when anchors are used 
- the use of codeListValue as mandatory can be introduced in KPI M, where we will check all references to codeLists

The uuidref would need further discussion, but this would introduce the use of unique identifiers that we would need to maintain , it's a big thing.  And given that we are allowing referencing multiple thesaurus from different communities then we would not have control over that.